### PR TITLE
Item table

### PR DESCRIPTION
**Pull request by bsonge**
#6 Added item table, takes in array of objects and makes a table out of them.

### CODE REVIEW

File: app/components/ItemTable/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    this.headerVals = Object.keys(this.props.data[0]);
+    this.header = this.headerVals.map((val) => <th>{val}</th>);
+    this.toDisplay = this.props.data.map((obj) => (
+      <tr>{ this.headerVals.map((val) => <td>{obj[val] || ''}</td>)}</tr>
+    )
+    );
+  }
+  render() {
+    return (
+      <Table>
```
Comment (Quinn-Donnelly) [inline app/components/ItemTable/index.js]: Add these options to the Table striped bordered condensed hover responsive

File: app/containers/HomePage/index.js [hunk 2]
```diff
@@ -11,14 +11,24 @@
...
+
 import messages from './messages';
 
+
 export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
+    this.data = [{ str: 'hi', num: 1234, bool: 'true', wild: 'bye' },
+                { str: 'aahsadpfajwepoifjawoipjfawjef', num: null, bool: 'false', wild: 420 },
+                { str: 'aah', num: null, bool: null, wild: null }];
+    this.thing = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa. Vestibulum lacinia arcu eget nulla. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. /n/nCurabitur sodales ligula in libero. Sed dignissim lacinia nunc. Curabitur tortor. Pellentesque nibh. Aenean quam. In scelerisque sem at dolor. Maecenas mattis. Sed convallis tristique sem. Proin ut ligula vel nunc egestas porttitor. Morbi lectus risus, iaculis vel, suscipit quis, luctus non, massa. Fusce ac turpis quis ligula lacinia aliquet. Mauris ipsum.';
```
Comment (Quinn-Donnelly) [inline app/containers/HomePage/index.js]: Clean testing components restore previous state.

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]

**Review by Quinn-Donnelly — APPROVED** [review]
Solid :100:

**Review by wrejcodes — APPROVED** [review]
Looks good

**Pull request merged**